docs/README.md

```diff
@@ -3,8 +3,8 @@ home: true
 heroImage: /regen-logo.png
 heroText: Regen Ledger 
 tagline: Technical docs for Regen Network's blockchain
-actionText: Get Started →
-actionLink: /getting-started/
+actionText: Introduction →
+actionLink: /intro/
```

Comment:
This updates the call to action button on the homepage. The "Getting Started" button in the top navbar will remain the same. I think it makes sense to only update the call to action on the homepage. Let me know if you think differently.

Reply:
Makes sense.